<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabOccupancyLiveViewSettings</class>
 <widget class="QWidget" name="TabOccupancyLiveViewSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="MyLabel" name="labelGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Group</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="MyLineEdit" name="lineEditMaxStays"/>
       </item>
       <item>
        <widget class="BlueLabel" name="label_10">
         <property name="text">
          <string>1~99999</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="MyLabel" name="labelGreenReminders">
       <property name="text">
        <string>Reminders of Green Light</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="MyLabel" name="labelNvrCountingReset">
       <property name="text">
        <string>NVR Counting Reset</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ComboBox" name="comboBoxGroup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="MyLabel" name="labelPeopleCounting">
       <property name="text">
        <string>People Counting</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="MyLabel" name="labelLiveViewCountingAutoReset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Live View Counting Auto Reset</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="MyLabel" name="labelRedReminders">
       <property name="text">
        <string>Reminders of Red Light</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="ComboBox" name="comboBoxFontSize"/>
     </item>
     <item row="11" column="0">
      <widget class="MyLabel" name="labelDay">
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ComboBox" name="comboBoxPeopleCounting"/>
     </item>
     <item row="11" column="1">
      <widget class="ComboBox" name="comboBoxDay"/>
     </item>
     <item row="12" column="0">
      <widget class="MyLabel" name="labelTime">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="MyLineEdit" name="lineEditGroupName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="MyTimeEdit" name="timeEditTime">
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="MyLabel" name="labelAlarmAction">
       <property name="text">
        <string>Alarm Action</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="MyLabel" name="labelGroupSettings">
       <property name="text">
        <string>Group Settings</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="MyLineEdit" name="lineEditGreenReminders">
       <property name="maxLength">
        <number>45</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="MyLineEdit" name="lineEditRedReminders">
       <property name="maxLength">
        <number>45</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="MyLabel" name="labelGroupName">
       <property name="text">
        <string>GroupName</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="ComboBox" name="comboBoxLiveViewCountingAutoReset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="MyLabel" name="labelFontSize">
       <property name="text">
        <string>Font Size</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="MyLabel" name="labelLiveViewCountingReset">
       <property name="text">
        <string>Live View Counting Reset</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="MyLabel" name="labelMaxStays">
       <property name="text">
        <string>Max. Stays</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="MyPushButton" name="pushButtonGroupSettings">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="MyPushButton" name="pushButtonLiveViewCountingReset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonNote1">
          <property name="styleSheet">
           <string notr="true">QToolButton
{
border: 0px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resource/resource.qrc">
            <normaloff>:/common/common/note.png</normaloff>:/common/common/note.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BlueLabel" name="labelNote1">
          <property name="text">
           <string>Reset the group counting data in live view.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="MyPushButton" name="pushButtonNvrCountingReset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonNote2">
          <property name="styleSheet">
           <string notr="true">QToolButton
{
border: 0px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resource/resource.qrc">
            <normaloff>:/common/common/note.png</normaloff>:/common/common/note.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="BlueLabel" name="labelNote2">
          <property name="text">
           <string>Reset the group counting data stored in NVR side.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="MyPushButton" name="pushButtonAlarmAction">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>38</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButtonBack">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MyTimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>mytimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>BlueLabel</class>
   <extends>QLabel</extends>
   <header>bluelabel.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
